### ISSUE TITLE
Rounding/Precision Errors

### ISSUE THREAD

**Issue by nicreuss** [svg2mod/svg2mod#36]
I've found the new default behavior of rounding to 10 decimal places in v0.1.6.3 introduces rounding errors in the coordinates of lines (possibly other shapes too but I haven't tested them).

For example, `25.5` became `24.4999999999` once I upgraded from v0.1.6.2 to v0.1.6.3.

What was the reasoning behind this modification and would it be possible to enable/disable this with a command-line flag?

**Comment by Sodium-Hydrogen** [comment]
I added the rounding behavior due to the algorithm to inline holes having some issues with floating point errors causing useless calculations. I choose 10 decimal places because that is already pretty precise for the units being worked with.

However, I have noticed that while faster, the inlining algorithm still isn't performing at a level that I like. The next update will add a better and more efficient hole inlining algorithm that will no longer suffer from the same floating point issues so I will be removing the rounding completely.
